### DISCUSSION TITLE
install question, copied the code to the terminal but nothing happened

### DISCUSSION THREAD

**Discussion by 1014493111** [requarks/wiki#3384] [Help / Questions]
**Question**
Describe in detail what you are trying to achieve and where do you need guidance.

**Host Info (please complete the following information):**
OS: [e.g. Ubuntu 18.04, Docker]
Wiki.js version: [e.g. 2.0.1]
Database engine: [e.g. postgres 9.7, mysql 5.7]

I am a novice of coding, I copied all the coding from the install guide to the terminal, I do not know is it because I got something wrong or else, nothing happened. Could you help me with this, please? thank you!

**Comment by Smankusors** [comment]
Which install guide did you use?

**Comment by winkelement** [comment]
> Question
> **Describe in detail** what you are trying to achieve and where do you need guidance.

I guess it's very hard for anyone to help you unless you **describe your problem in detail**.

>  I copied all the coding

What exactly did you copy into the terminal?

> nothing happened

That is very unlikely, unless you did not press enter. What is the output in the terminal?

And of course see above:

> Which install guide did you use?
